GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/MainWindowViewModel.cs

```diff
@@ -2,10 +2,12 @@
 using GalaxyZooTouchTable.Utility;
 using PanoptesNetClient;
 using PanoptesNetClient.Models;
+using System;
```

Comment:
You should probably avoid importing all of `System`

Reply:
I don't think importing all of `System` affects performance, but I've removed it for posterity.